src/plugins/intel_gpu/src/plugin/ops/strided_slice.cpp

```diff
@@ -227,9 +227,14 @@ static void CreateStridedSliceOp(Program& p, const std::shared_ptr<ngraph::op::v
 
         // Reshape in case of deleting of axis
         if (!shrink_axis_mask.empty()) {
-            auto targetShape = tensor_from_dims(output_shape);
+            std::vector<int64_t> output_pattern(output_shape.size());
+            auto out_p = output_pattern.begin();
+            for (auto s = output_shape.begin(); s != output_shape.end() && out_p != output_pattern.end(); s++, out_p++) {
+                *out_p = *s;
+            }
+
             auto reshapeOutName = op->get_friendly_name() + "/Crop";
-            auto reshapePrim = cldnn::reshape(reshapeOutName, layerName, targetShape);
+            auto reshapePrim = cldnn::reshape(reshapeOutName, layerName, false, output_pattern, output_pshape);
```

Comment:
I am afraid this might cause error in static shape case. Could you trigger benchmark validation? Also could you pass the strided slice gpuFunc tests?

Reply:
This code is for only for static shape. If the output_pshape(origin of output_shape) is dynamic, this code never be reached. See line 38: if (input_pshape.is_dynamic() || output_pshape.is_dynamic())

Reply:
I think you can just trigger the dlb CI with compare graphs mode (Default mode) to check whether there is affected networks